### DISCUSSION TITLE
TwitchTag.is_auto is not a bool.

### DISCUSSION THREAD

**Discussion by cafce25** [twitch-rs/twitch_api#107] [Q&A]
There is certainly something i'm overlooking but I was very surprised when I found out [TwitchTag.is_auto](https://github.com/Emilgardis/twitch_api2/blob/b7d7015c65b3f33ce5a3f02d49a8fcde4e75962f/src/helix/tags.rs#L77) is not a bool but a custom enum type. I would like to know how a custom boolean type is advantageous here.

**Comment by Emilgardis** [comment]
I don't think there is much advantage, except for it making it more obvious what type of truth it signifies.
